updater/internal/path/path_linux.go

```diff
@@ -23,3 +28,36 @@ const LinuxInstallDir = "/opt/observiq-otel-collector"
 func InstallDir(_ *zap.Logger) (string, error) {
 	return LinuxInstallDir, nil
 }
+
+// LinuxServiceCmdName returns the filename of the service command available
+// on this Linux OS. Will be one of systemctl and service
+func LinuxServiceCmdName() string {
+	var path string
+	var err error
+	path, err = exec.LookPath("systemctl")
+	if err != nil {
+		path, err = exec.LookPath("service")
+	}
+	if err != nil {
+		// Defaulting to systemctl in the most common path
+		// This replicates prior behavior where
+		path = "/usr/bin/systemctl"
+	}
+	_, filename := filepath.Split(path)
+	return filename
+}
+
+// LinuxServiceFilePath returns the full path to the service file
+func LinuxServiceFilePath() string {
+	var path string
+	var err error
+	path, err = exec.LookPath("/usr/lib/systemd/system/observiq-otel-collector.service")
```

Comment:
LookPath looks specifically for executable files, so I think this will never return a path for the systemd service file.

I think instead we should just hard-code the path based on the detected init system, which this function can just take in as a parameter.

Reply:
The shell scripts are executable, and this should work from my understanding. However, if you still think it should be changed that should be pretty quickly handled.

Reply:
I'm talking specifically about the systemd service file here.
```sh
$ sudo ls -la /usr/lib/systemd/system/observiq-otel-collector.service
-rw-r--r-- 1 root root 688 Mar 12 12:41 /usr/lib/systemd/system/observiq-otel-collector.service
```

We should either:
1. Hardcode the path based on init system OR
2. Use os.Stat to check for file existence, it's more suitable for this check than exec.LookPath